hack/lldb/etc/sysctl.conf

```diff
@@ -0,0 +1,2 @@
+net.ipv6.conf.all.disable_ipv6 = 1
```

Comment:
Why out of curiosity?

Reply:
I had a weird issue where the lldb connection was trying to use the ipv6 address and failing, this fixed it. I have ipv6 turned off on my host.